tests/org.eclipse.n4js.spec.examples.xpect.tests/testdata/my/Ex65_Object_Literal.n4js.xt

```diff
@@ -57,7 +57,7 @@ stderr:
 			}
 		};
 	});
-})(typeof module !== 'undefined' && module.exports ? require('n4js-node/index').System(require, module) : System);
+})(typeof module !== 'undefined' && module.exports ? require('n4js-node/src-gen/index').System(require, module) : System);
```

Comment:
You changed several spec example tests. But it seems you did not update the examples in the spec accordingly?

Reply:
in spec we have only parts of the code from tests. Parts I have changed are not there.